cmd/textiled/main.go

```diff
@@ -253,9 +253,9 @@ var rootCmd = &cobra.Command{
 		addrGatewayHost := cmd.AddrFromStr(configViper.GetString("addr.gateway.host"))
 		addrGatewayUrl := configViper.GetString("addr.gateway.url")
 
-		var addrFilecoinApi ma.Multiaddr
-		if str := configViper.GetString("addr.filecoin.api"); str != "" {
-			addrFilecoinApi = cmd.AddrFromStr(str)
+		var addrPowergateApi ma.Multiaddr
+		if str := configViper.GetString("addr.powergate.api"); str != "" {
+			addrPowergateApi = cmd.AddrFromStr(str)
```

Comment:
This config is optional. If not set, archive feature isn't available... so you're not forced to have a PG instance running.